Add backup and restore content for tiup-cluster.md (#10907)
pingcap/docs: tiup/tiup-cluster.md

> **Note:**
>
> It is recommended that you back up the `.tiup` directory regularly to avoid the loss of TiUP data caused by abnormal conditions, such as disk damage of the control machine.

## Back up and restore meta files for cluster deployment and O&M

If the meta files used for operation and maintenance (O&M) are lost, managing the cluster using TiUP will fail. It is recommended that you back up the meta files regularly by running the following command:

```bash
tiup cluster meta backup ${cluster_name}
```

If the meta files are lost, you can restore them by running the following command:

```bash
tiup cluster meta restore ${cluster_name} ${backup_file}
```

> **Note:**
>

> > The restore operation overwrites the current meta files. Therefore, perform a restore operation only when the meta files are lost.

Reviewer: ```suggestion
> The restore operation overwrites the current meta files. Therefore, it is recommended to restore only when the meta files are lost.
```
